ee/clickhouse/queries/clickhouse_retention.py

```diff
@@ -61,8 +61,16 @@ def calculate_retention(self, filter: Filter, team: Team, total_intervals: int)
             ),
             {
                 "team_id": team.pk,
-                "start_date": date_from.strftime("%Y-%m-%d %H:%M:%S"),
-                "end_date": date_to.strftime("%Y-%m-%d %H:%M:%S"),
+                "start_date": date_from.strftime(
```

Comment:
ee/clickhouse/queries/utils.py already has this code.

Extract method into utils? `format_range_date(filter, date)`?

Reply:
just had to change it because retention uses the period paramtere not interval so it's slightly diff

Reply:
Ah, makes more sense then.